### PR TITLE
Added cv docs

### PR DESCRIPTION
**Pull request by Erol444**
Feature tracker gif
feature_tracker

### CODE REVIEW

**Comment by Erol444** [comment]
@szabi-luxonis  is there anything else to add under these CV docs?

File: source/pages/cv.rst [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+   tutorials/creating-custom-nn-models.rst
+
+Our platform supports computer vision (CV) functions to be performed on the device itself. While you can't run `OpenCV <https://github.com/opencv/opencv>`__,
+you can use many of its supported functions. With DepthAI pipeline builder you can:
+
+- **Crop, rotate, warp/dewarp, mirror, flip, transform perspective**, etc. with `ImageManip <https://docs.luxonis.com/projects/api/en/latest/components/nodes/image_manip/>`__ node
+- **Detect edges** (Sobel filter) with `EdgeDetector <https://docs.luxonis.com/projects/api/en/latest/components/nodes/edge_detector/>`__ node
+- **Detect and track features** with `FeatureTracker <https://docs.luxonis.com/projects/api/en/latest/components/nodes/feature_tracker/>`__ node
+- **Track objects** (Kalman filter, Hungarian algorithm) with `ObjectTracker <https://docs.luxonis.com/projects/api/en/latest/components/nodes/object_tracker/>`__ node. Currently supported Yolo and MobileNet object detectors.
```
Comment (szabi-luxonis) [inline source/pages/cv.rst:15]: Technically you can feed any bounding box to it, needs to be `ImgDetections`

File: source/pages/cv.rst [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+.. toctree::
+   :hidden:
+
+   tutorials/creating-custom-nn-models.rst
+
+Our platform supports computer vision (CV) functions to be performed on the device itself. While you can't run `OpenCV <https://github.com/opencv/opencv>`__,
+you can use many of its supported functions. With DepthAI pipeline builder you can:
+
+- **Crop, rotate, warp/dewarp, mirror, flip, transform perspective**, etc. with `ImageManip <https://docs.luxonis.com/projects/api/en/latest/components/nodes/image_manip/>`__ node
+- **Detect edges** (Sobel filter) with `EdgeDetector <https://docs.luxonis.com/projects/api/en/latest/components/nodes/edge_detector/>`__ node
```
Comment (szabi-luxonis) [inline source/pages/cv.rst:13]: StereoDepth is CV too, can be cross linked.

**Review by cafemoloko — APPROVED** [review]
LGTM

**Review by szabi-luxonis — APPROVED** [review]

**Pull request merged**